src/sass/language_base.scss

```diff
@@ -3,4 +3,17 @@
 	.ja-hide{
 		display: none!important;
 	}
-}
\ No newline at end of file
+	.ja-show{
+		display: inline!important;
+		visibility: visible!important;
+	}
+	#header #menu-top li a.smaller-font, #header ul#menu-top.smaller-font{
+		font-size: 90%;
+	}
+	@media screen and (max-width: 845px) {
```

Comment:
Because the links break and disappear at this exact point. You can't see
the header links for get started, trade, etc anymore
On 27 Aug 2015 13:39, "Raunak Kathuria" <notifications@github.com> wrote:

> In src/sass/language_base.scss
> <https://github.com/binary-com/binary-static/pull/405#discussion_r38063759>
> :
>
> > \ No newline at end of file
> > +	.ja-show{
> > +		display: inline!important;
> > +		visibility: visible!important;
> > +	}
> > +	#header #menu-top li a.smaller-font, #header ul#menu-top.smaller-font{
> > +		font-size: 90%;
> > +	}
> > +	@media screen and (max-width: 845px) {
>
> why 845px was chosen? standards are generally 768px/640px
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/binary-com/binary-static/pull/405/files#r38063759>.
>